Made db seeding script. Used script to generate 100 random docs.

Kestrad opened the pull request:
@Quanquest Please review my seeding script!

Kestrad committed:
Made db seeding script. Used script to generate 100 random docs.

Quanquest reviewed (commented):
Diana, your Python looks great.  How does this get implemented to the database?

Kestrad committed:
Added script that actually seeds database

npm run seed will now execute db/seed.js, which runs the python
script and then grabs that generated file and adds it to the db

Kestrad commented:
@Quanquest 

I've now actually added a js script that actually seeds the db!
